Chore(Docs): Fixed broken links and readability

SSmale opened the pull request:
Checked:
- [x] about.md
- [x] components.md
- [x] install.md
- [x] index.md
- [x] trivia_cloning.md

SSmale committed:
Chore(Docs): Removed broken link

Removed the broken link to the Advanced documentation and related text

SSmale committed:
Chore(Docs): Changed some broken english

Changed the sentence to make it read a little better
Removed may to as the bot needs to be started to continue

SSmale committed:
Docs: Forgot to save file after making a chnage on QA

SSmale committed:
Chore(Docs): Fixed broken links

Fixed the order of the words and fixed the english

Fixed broken links to LUIS and botpress-nlu package

SSmale committed:
Chore(Docs):

 - Removed unecessary text

 - Tutorial folder no longer contains each step folder
so removed this to save confusion

 - Gave the actual line from the bot to say it had started

 - Added links to Slack and Github Issues should there be any problems

 - Started to amend the text.
Will think further on this to make it read better

slvnperron commented on docs/getting_started/trivia_template.md:
````diff
@@ -88,11 +86,15 @@ npm start
+```bash 
+info: Bot launched. Visit: http://localhost:3000
+``` 
+Should you get an error, please visit our [Slack channel](https://slack.botpress.io) or [open an issue](https://github.com/botpress/botpress/issues/new).
````
Let's point the community to `https://help.botpress.io` instead of Slack going forward as we are trying to transition Slack to the Forum :)

epaminond replied:
@SSmale

slvnperron commented:
Great!!

SSmale committed:
Chore(Docs): Change signposting to the forum

 - Changed both locations mentioning Slack to the help forum

 - Better signposting to
   - search and then ask in the forum
   - got to GitHub if the issue is code related

SSmale committed:
Merge branch 'master' into master

SSmale committed:
Merge remote-tracking branch 'upstream/master'

SSmale committed:
Pulled from upstream

SSmale commented:
That is the changes made to those docs, they now direct to the forum @slvnperron @epaminond
